components/autofill/uniffi.toml

```diff
@@ -1,6 +1,5 @@
 [bindings.kotlin]
 package_name = "mozilla.appservices.autofill"
-cdylib_name = "megazord"
```

Comment:
I am curious: Why can we remove the `cdylib_name`?

Reply:
That's one perk of library mode, since we're passing the library path to the bindings generator we can calculate the cdylib name from that.